src/Akka.Monitoring.PerformanceCounters.Demo/Program.cs

```diff
@@ -34,8 +34,28 @@ protected override void PreStart()
         }
     }
 
-    class HelloActor : TypedActor, IHandle<string>
+    class HelloActor : ActorBase
```

Comment:
I've replaced all `TypedActor`s, because we already marked them as obsolete and will get rid of them in the future.